src/spec.ts

```diff
@@ -103,6 +104,11 @@ export interface GenericUnitSpec<E extends Encoding<any>, M> extends BaseSpec, L
    */
   encoding: E;
 
+  /**
+   * A geo projection
```

Comment:
An object defining properties of geographic projection.

Please also describe more if this can be null or something? 
Does it work only for certain type of marks? (If all marks, ignore my comment.) 

What's the `__Default value:__`, is there any case it is generated by default?